statman/Poisson/Poissondist.py

```diff
@@ -0,0 +1,141 @@
+import math
+from scipy.stats import poisson
+from random import seed
+from ..Generaldistribution import statman
+
+# import and setup seaborn
+import seaborn as sns
+# settings for seaborn plotting style
+sns.set(color_codes=True)
+# settings for seaborn plot sizes
+sns.set(rc={'figure.figsize': (5, 5)})
+
+
+class Poisson(statman):
+    """
+        Poisson distribution class for calculating and
+        visualizing a Poisson distribution
+
+        Attributes:
+            mean (float) representing the mean value of the distribution,
+            variance/var (float) representing the variance of the distribution,
+            e (float) representing Euler's number,
+            p (float) representing the probability of an occurance
+            x (int) number of occurrences
+    """
+
+    def __init__(self, mu=0):
+
+        statman.__init__(self)
+
+        # constant
+        self.e = 2.718282
+
+    def calculate_mean(self):
+        """
+            Function to get the mean of the data set.
+
+            Args:
+                None
+
+            Returns:
+                float: the mean value,
+                float: the kurtosis value.
+        """
+
+        mean, variance, skew, kurt = poisson.stats(mu, moments='mvsk')
+
+        self.mean = mean
+        self.variance = variance
+        self.kurt = kurt
+
+        return mean, kurt, variance
```

Comment:
Since the calculate_mean method calculates the values of not only the mean, we should either change the name of the function or separate the methods into standalones